ASAManager/StatusTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net;
+using Helpers.Http;
+using Xunit;
+
+namespace AsaManager
+{
+    public class StatusTest
+    {
+        private readonly IHttpClient httpClient;
+        private const string ASA_MANAGER_ADDRESS = "http://localhost:9024/v1";
```

Comment:
nit: this value will be used across all the tests, so it might be worth moving it to `helpers/Constants.cs` now

Reply:
I will take care of this in my next PR, going to move this and others to a settings file like in StorageAdapter tests.

Reply:
I'd use this PR to sync the versions, using a stable one. Might be worth checking if the test guidelines have a note about this, where talking about flaky tests

Reply:
nit: can this address be loaded from consolidated list in 'env.list' or from environment so that it is not hard coded?

Reply:
I will take care of constants in next PR (this one is just to get setup done). I will unify versions in this PR.